# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/public/epplets-extra16/e-mount.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- E-Mount_0.3/Makefile.orig-rock-maint	2003-08-29 09:31:58.000000000 +0200
+++ E-Mount_0.3/Makefile	2003-08-29 09:32:49.000000000 +0200
@@ -2,13 +2,13 @@
 
 IMLIB_CFLAGS = `imlib-config --cflags`
 IMLIB_LIBS = `imlib-config --libs`
-GCCOPTS = -MMD -Wall
+GCCOPTS = -MMD -Wall $(CFLAGS)
 GCC = gcc
 
 all : E-Mount.epplet
 
 E-Mount.epplet : E-Mount.c
-	${GCC} ${GCCOPTS}  E-Mount.c -o E-Mount.epplet -lepplet ${IMLIB_LIBS} ${IMLIB_CFLAGS}
+	${GCC} ${GCCOPTS}  E-Mount.c -o E-Mount.epplet -L$(E16)/lib -lepplet ${IMLIB_LIBS} ${IMLIB_CFLAGS} -lGL
 
 install : E-Mount.epplet
 	install -m 755 E-Mount.epplet ${EBIN}/E-Mount.epplet
--- E-Mount_0.3/E-Mount.c.orig	2004-09-13 10:10:10.875988192 +0200
+++ E-Mount_0.3/E-Mount.c	2004-09-13 10:10:24.083980272 +0200
@@ -225,6 +225,7 @@
 			changelabel = 0;
 			break;
 		default:
+			;
 	}
 
 	updatefsinfo ((*pointertocurrentlistitem)->mountpoint, changelabel);
